From 1c61cd3e2d107bd79760a37d69cd164204a21538 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Aug 2018 13:48:16 +0300
Subject: [PATCH] version-hooks: add new package

The version-hooks package is not a real package, but an addition
to habanalabs external buildroot.
It helps in propagating the git versions to the rsync'ed source
code which does not have the .git directory.
Currently enable only for U-Boot.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
new file mode 100644
index 0000000..4a8ab1b
--- /dev/null
+++ b/package/version-hooks/version-hooks.mk
@@ -0,0 +1,10 @@
+define HL_VERSION_GEN
+	if [ -n $(SRCDIR) ]; then \
+		pushd $(SRCDIR); \
+		echo -n "-`git describe 2>/dev/null`" > $(@D)/localversion-habanalabs; \
+		popd; \
+	fi
+endef
+
+UBOOT_POST_RSYNC_HOOKS += HL_VERSION_GEN
+
